subprojects/core-api/src/main/java/org/gradle/api/tasks/util/PatternSet.java

```diff
@@ -174,6 +176,19 @@ public Spec<FileTreeElement> getAsExcludeSpec() {
         return patternSpecFactory.createExcludeSpec(this);
     }
 
+    /**
+     * Like {@link #getIncludes()}, but returns an immutable set.
```

Comment:
:x: To me immutable suggests that it can't change, but it returns just an `unmodifiableSet` view of the underlying set, so it can still fail if the source is modified. Or, am I missing something?

Reply:
At places this is generally used it is supposed to be no longer modified, but given that it's possible I agree it should state "view" instead.